readme.md

```diff
@@ -20,3 +28,9 @@ This config follows the spirit of the jQuery [code style](https://contribute.jqu
 ## Semver policy
 
 Same approach as in ESLint, see https://github.com/eslint/eslint#user-content-semantic-versioning-policy.
+
+## Testing
+
+To avoid a circular dependency, testing copies JQuery source code into the test directory before running linting for
```

Comment:
`jquery` would be just a `devDependency`, I don't see a circular dependency here. Am I missing anything?

Reply:
This project is one of jquery's devDependencies -- see [package.json](https://github.com/jquery/jquery/blob/19716254877870ecd649272cadd00a0d0ff8be01/package.json#L126)

Reply:
As per your other comment in copy-fixtures.js, I am adding it to devDependencies and removing this note.